src/js/vendor/three/examples/js/libs/dat.gui.min.js

```diff
@@ -11,4 +11,4 @@
  * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Is this file used by IK?

Reply:
No, i've used dat.gui in test project but i didn't transfer it to the storyboarder.